CodeForBaltimore/Bmore-Responsive
# Adding sequelize cli

Adding sequelize cli will make seeding much easier. This also removes the old
method of seeding and deleting the db.

## Todos

- [x] `.env` use for seeding removed
- [x] seeder scripts added to `package.json` 
- [x] document new processes
- [x] get Docker working again!

## Steps to Test or Reproduce
Bring up the db and app as normal. Make sure you start with a fresh db: no tables! If you already have tables no worries, this should override it...
```sh
npm run db-create
npm run db-seed
npm run db-unseed
npm run db-delete
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

- seeding
